README.md

```diff
@@ -1,24 +1,220 @@
-# README
+# Spree Starter
```

Comment:
copied from https://github.com/spree/spree_starter ?

didn't we start our on rails 8 project and only add spree related gems?

Reply:
this file seem to have too much of noise, and things we dont practice at our company